source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm

```diff
@@ -133,6 +136,17 @@ - (ACRHorizontalAlignment)horizontalContentAlignment
     return ACRLeft;
 }
 
+- (ACRContainerStyle)style
```

Comment:
>style [](http://example.com/codeflow?start=21&length=5)

rename to getLastStyle or peekStyleContext or something similar

Reply:
@almedina-ms, this is a getter for the property style. for setter of a property, set + property name.